src/util/log.c

```diff
@@ -15,6 +16,34 @@
 #define unlikely(x)     (x)
 #endif
 
+static int caseInsensitiveStrncmp(char const * const string1, 
```

Comment:
Could you please add some documentation for this function, esp regarding the reason for us to reimplement strcasecmp ?
Thanks